pkg/webreg/webreg.go

```diff
@@ -1966,6 +1967,94 @@ const staticBoskosConfig = `resources:
   - "server03.prod.service.com"
 `
 
+const linksPage = `<h2 id="clusters"><a href="#clusters">Clusters</a></h2>
+<p>The clusters that currently comprise CI are:</p>
+<ul>
+  <li>
+    <a href="https://console.svc.ci.openshift.org"><code>api.ci</code></a>:
+    legacy Openshift 3.11 cluster in GCP.  Job execution is being migrated out
+    of it.
+  </li>
+  <li>
+    <a href="https://console-openshift-console.apps.ci.l2s4.p1.openshiftapps.com"><code>app.ci</code></a>:
+    Openshift Dedicated 4.x cluster containing most Prow services.
+  </li>
+  <li>
+    <a href="https://console-openshift-console.apps.build01.ci.devcluster.openshift.com/"><code>build01</code></a>:
+    Openshift 4.x cluster in AWS that executes a growing subset of the jobs.
+  </li>
+</ul>
+<p>
+These clusters use Github OAuth authentication: all members of the Openshift
+organization in Github can log in.
+</p>
+<h2 id="services"><a href="#services">Services</a></h2>
+<p>Below is a non-exhaustive list of CI services.</p>
+<ul>
+  <li>
+    <a href="https://prow.ci.openshift.org">https://prow.ci.openshift.org</a>:
+    main Prow dashboard with information about jobs, pull requests, the merge
+    queue, etc.
+  </li>
+  <li>
+    <a href="https://openshift-release.svc.ci.openshift.org">
+      https://openshift-release.svc.ci.openshift.org
+    </a>: OCP release status page.
+  </li>
+  <li>
+    <a href="https://origin-release.svc.ci.openshift.org">
+      https://origin-release.svc.ci.openshift.org
+    </a>: OKD release status page.
+  </li>
+  <li>
+    <a href="https://search.ci.openshift.org">https://search.ci.openshift.org</a>:
+    search tool for error messages in job logs and Bugzilla bugs.
+  </li>
+  <li>
+    <a href="https://sippy.ci.openshift.org">https://sippy.ci.openshift.org</a>:
+    CI release health summary.
+  </li>
+</ul>
+<h2 id="contact"><a href="#contact">Contact</a></h2>
+<p>DPTP maintains several means of contact:</p>
+<ul>
+  <li>
+    Slack
+    <ul>
+      <li>
+        <code>#announce-testplatform</code>: general announcements and outages.
+        Usage is limited to the DPTP team, please do not post messages there.
+      </li>
+      <li>
+        <code>#forum-testplatform</code>: general queries and discussion for
+        the test platform.
```

Comment:
...test platform. For general assistance, ping @dptp-helpdesk. For reporting an outage, ping @dptp-triage.